### PR TITLE
Colon in claim name and JSON type claim value

### PR DESCRIPTION
**Pull request by colcek**
As an example, AWS Cognito JWT token contains `"cognito:groups": ["group1"]` as claim and it is also possible to see JSON object as claim values.

Code is updated to support following ENV var esp. docker config:
`CLAIM_MAPPINGS: "sub:x-user-id,cognito:groups:x-user-groups,username:x-user-name"`

In addition, any JSON parsable value is forwarded as JSON string in the mapped header key

### CODE REVIEW

File: decoder/jws.go [hunk 1]
```diff
@@ -51,7 +52,13 @@ func (d *jwsDecoder) Decode(ctx context.Context, rawJws string) (*Token, error)
 			if strVal, ok := value.(string); ok {
 				token.Claims[destKey] = strVal
 			} else {
-				return nil, UnexpectedClaimTypeError{key, value}
+				strJson, err := json.Marshal(value)
```
Comment (SimonSchneider) [inline decoder/jws.go]: the linter is complaining that this should be `strJSON`, would you mind changing it?

**Review by SimonSchneider — CHANGES_REQUESTED** [review]
Looks great, thank you for this!

If you could take care of the one linter issue I also hope that another commit would trigger a pipeline run as I'm unsure to why Github actions decided not to run for this PR.

File: decoder/jws.go (hunk 1 above)
Comment (colcek) [reply decoder/jws.go]: Fixed the linter warning (sorry it looks I forgot to enable `golint`)

Comment (SimonSchneider) [reply decoder/jws.go]: Thank you!

**Comment by SimonSchneider** [comment]
Looks good to me I've gone ahead and excluded the actions pass requirement as it seems they don't get triggered on forks. Might be something to look into in future.

**Comment by SimonSchneider** [comment]
Thank you for this @colcek the new version is available now: https://hub.docker.com/r/simonschneider/traefik-jwt-decode

**Pull request merged**